src/routes.js

```diff
@@ -94,18 +94,29 @@ module.exports = [
     handler: (request, reply) => {
       var data =  {
         prev: '/sleep',
-        next: '/finished'
+        next: '/school'
       };
       reply.view('friends', data);
     }
   },
   {
     method: 'GET',
+    path: '/school',
+    handler: (request, reply) => {
+      var data =  {
```

Comment:
Your tests want to be checking if this data is having the right effect on the template